src/AC/ERegBoard.cpp

```diff
@@ -16,43 +16,8 @@ ERegBoard::ERegBoard(HardwareSerial *serial, uint8_t id) {
     failPacket_ = {.id = 255};
 }
 
-void ERegBoard::sendSerial(Comms::Packet *packet) {
-    uint32_t timestamp = millis();
-    packet->timestamp[0] = timestamp & 0xFF;
-    packet->timestamp[1] = (timestamp >> 8) & 0xFF;
-    packet->timestamp[2] = (timestamp >> 16) & 0xFF;
-    packet->timestamp[3] = (timestamp >> 24) & 0xFF;
-
-    uint16_t checksum = computePacketChecksum(packet);
-    packet->checksum[0] = checksum & 0xFF;
-    packet->checksum[1] = checksum >> 8;
-
-    for (int i = 0; i < 3; i++) {
-        serial_->write(packet->id);
-        serial_->write(packet->len);
-        serial_->write(packet->timestamp, 4);
-        serial_->write(packet->checksum, 2);
-        serial_->write(packet->data, packet->len);
-        serial_->write(0x68);
-        serial_->write(0x69);
-        serial_->write(0x70);
-    }
-}
-
-Comms::Packet *ERegBoard::receiveSerial() {
-    if(serial_->available()) {
-        packetBufferCtr_ = 0;
-            while(serial_->available() && packetBufferCtr_ < sizeof(Comms::Packet)) {
-                packetBuffer_[packetBufferCtr_] = serial_->read();
-                packetBufferCtr_++;
-            }
-            Comms::Packet *packet = (Comms::Packet *)packetBuffer_;
-            // DEBUG("Got unverified packet with ID ");
-            // DEBUG(packet->id);
-            // DEBUG('\n');
-            return packet;
-    }
-    return &failPacket_;
+void ERegBoard::sendEthernet(Comms::Packet *packet, uint8_t ip) {
```

Comment:
you have to add the checksum and timestamp before sending the packet, see emitPacket in comms for how to do that